### ISSUE TITLE
Questions about the dataloader in 'helper_functions.py'.

### ISSUE THREAD

**Issue by Yijing-c** [Alibaba-MIIL/ML_Decoder#7]
The code in 'helper_functions.py':
` output = torch.zeros((3, 80), dtype=torch.long)
  for obj in target:
      if obj['area'] < 32 * 32:
          output[0][self.cat2cat[obj['category_id']]] = 1
      elif obj['area'] < 96 * 96:
          output[1][self.cat2cat[obj['category_id']]] = 1
      else:
          output[2][self.cat2cat[obj['category_id']]] = 1
  target = output`

Because I want to try this on another dataset, I am wondering that why you use the area's size of object to build the category metric? 
Thank you for your general sharing! And hope for your more guidance.

**Comment by mrT23** [comment]
This line was taken from COCO official implementation.
i think it has something to do with object detection (small vs large objects).

in any case, we are not using it in multi-label classification:
https://github.com/Alibaba-MIIL/ML_Decoder/blob/main/train.py#L103

you can remove it when using another dataset.
